doc/spire_server.md

```diff
@@ -68,6 +68,7 @@ This may be useful for templating configuration files, for example across differ
 | `ratelimit`                 | Rate limiting configurations, usually used when the server is behind a load balancer (see below)  |                                                                |
 | `socket_path`               | Path to bind the SPIRE Server API socket to                                                       | /tmp/spire-server/private/api.sock                             |
 | `trust_domain`              | The trust domain that this server belongs to (should be no more than 255 characters)              |                                                                |
+| `auth_policy_engine`        | The [auth policy engine](doc/authorization_policy_engine.md) used for authorization decision      | default SPIRE authorization policy                             |
```

Comment:
This document is already in `/doc` so a relative reference to `doc` will link to /doc/doc.

```suggestion
| `auth_policy_engine`        | The [auth policy engine](/doc/authorization_policy_engine.md) used for authorization decisions      | default SPIRE authorization policy                             |
```